lib/puppet/provider/gcompute_forwarding_rule/google.rb

```diff
@@ -124,6 +124,7 @@ def flush
     debug('flush')
     # return on !@dirty is for aiding testing (puppet already guarantees that)
     return if @created || @deleted || !@dirty
+    target_update(@resource) if @dirty[:target]
     update_req = Google::Compute::Network::Put.new(self_link(@resource),
```

Comment:
Forwarding rules don't support a full update, so here you'll want to only do the partial